dashboard/app/models/reviewable_project.rb

```diff
@@ -0,0 +1,19 @@
+# == Schema Information
+#
+# Table name: reviewable_projects
+#
+#  id             :bigint           not null, primary key
+#  storage_app_id :integer          not null
+#  user_id        :integer          not null
+#  level_id       :integer          not null
+#  script_id      :integer          not null
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#
+# Indexes
+#
+#  index_reviewable_projects_on_storage_app_id_and_user_id  (storage_app_id,user_id)
+#
+class ReviewableProject < ApplicationRecord
+  acts_as_paranoid
```

Comment:
This is only relevant if you have a `deleted_at` column and want to "soft delete" rows by just populating the `deleted_at` field instead of actually deleting the row. Check out this gem for more details:

https://github.com/rubysherpas/paranoia

Reply:
Ah gotcha, thanks - will remove.